### ISSUE TITLE
staging unuseable

### ISSUE THREAD

**Issue by paregorios** [isawnyu/pleiades-gazetteer#168]
I cannot presently test stories 34 and 51 because most terms have been removed from the time periods vocabulary, such that all the place pages I tried to visit elicit errors such as this:

```
Traceback (innermost last):

Module ZPublisher.Publish, line 138, in publish
Module ZPublisher.mapply, line 77, in mapply
Module ZPublisher.Publish, line 48, in call_object
Module Shared.DC.Scripts.Bindings, line 322, in __call__
Module Shared.DC.Scripts.Bindings, line 359, in _bindAndExec
Module Products.CMFCore.FSPageTemplate, line 237, in _exec
Module Products.CMFCore.FSPageTemplate, line 177, in pt_render
Module Products.PageTemplates.PageTemplate, line 87, in pt_render
Module zope.pagetemplate.pagetemplate, line 132, in pt_render
Warning: Macro expansion failed
Warning: <type 'exceptions.KeyError'>: 'macro'
Module zope.pagetemplate.pagetemplate, line 240, in __call__
Module collective.newrelic.patches.talinterpreter, line 17, in monkeypatch
Module newrelic.api.function_trace, line 110, in literal_wrapper
Module zope.tal.talinterpreter, line 271, in __call__
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 888, in do_useMacro
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 533, in do_optTag_tal
Module zope.tal.talinterpreter, line 518, in do_optTag
Module zope.tal.talinterpreter, line 513, in no_tag
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 954, in do_defineSlot
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 533, in do_optTag_tal
Module zope.tal.talinterpreter, line 518, in do_optTag
Module zope.tal.talinterpreter, line 513, in no_tag
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 858, in do_defineMacro
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 954, in do_defineSlot
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 533, in do_optTag_tal
Module zope.tal.talinterpreter, line 518, in do_optTag
Module zope.tal.talinterpreter, line 513, in no_tag
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 954, in do_defineSlot
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 533, in do_optTag_tal
Module zope.tal.talinterpreter, line 518, in do_optTag
Module zope.tal.talinterpreter, line 513, in no_tag
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 946, in do_defineSlot
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 533, in do_optTag_tal
Module zope.tal.talinterpreter, line 518, in do_optTag
Module zope.tal.talinterpreter, line 513, in no_tag
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 858, in do_defineMacro
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 533, in do_optTag_tal
Module zope.tal.talinterpreter, line 518, in do_optTag
Module zope.tal.talinterpreter, line 513, in no_tag
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 888, in do_useMacro
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 533, in do_optTag_tal
Module zope.tal.talinterpreter, line 518, in do_optTag
Module zope.tal.talinterpreter, line 513, in no_tag
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 852, in do_condition
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 531, in do_optTag_tal
Module zope.tal.talinterpreter, line 513, in no_tag
Module zope.tal.talinterpreter, line 343, in interpret
Module zope.tal.talinterpreter, line 742, in do_insertStructure_tal
Module Products.PageTemplates.Expressions, line 218, in evaluateStructure
Module zope.tales.tales, line 696, in evaluate
URL: file:/srv/python27-apps/pleiades4/src/Products.PleiadesEntity/Products/PleiadesEntity/skins/PleiadesEntity/place_view.pt
Line 159, Column 12
Expression: <PathExpr standard:u'context/@@names-listing'>
Names:
{'container': <Place at /plone/places/927604602>, 'context': <Place at /plone/places/927604602>, 'default': <object object at 0x7f0d03520bb0>, 'here': <Place at /plone/places/927604602>, 'loop': {}, 'nothing': None, 'options': {'args': ()}, 'repeat': <Products.PageTemplates.Expressions.SafeMapping object at 0x7f0cdc0da838>, 'request': <HTTPRequest, URL=http://pleiades.jazkarta.com/places/927604602/base_view>, 'root': <Application at >, 'template': <FSPageTemplate at /plone/places/927604602/base_view>, 'traverse_subpath': [], 'user': <PloneUser 'thomase'>}
Module zope.tales.expressions, line 217, in __call__
Module Products.PageTemplates.Expressions, line 155, in _eval
Module Products.PageTemplates.Expressions, line 117, in render
Module Products.PleiadesEntity.browser.attestations, line 80, in __call__
Module Products.PleiadesEntity.browser.attestations, line 20, in timeSpan
Module Products.PleiadesEntity.content.Temporal, line 126, in temporalRange
KeyError: 'hellenistic-roman-early-empire'
```

**Comment by paregorios** [comment]
Here's what one sees presently for the time periods vocab in pleiades settings:

<img width="933" alt="screen shot 2016-06-17 at 8 22 26 am" src="https://cloud.githubusercontent.com/assets/263285/16155532/b904dd5c-3464-11e6-8220-3900297c0386.png">

**Comment by davisagli** [comment]
Sorry, this was due to a failure in coordination and should be fixed now.

Carlos adjusted the setup code in pleiades.vocabulary to import some test data to the new vocab to fix tests, but I didn't notice that it was doing that even if the vocab already had values. So I ran the setup to add Witek's vocabulary, not realizing that it would wipe out the time periods. Fortunately since no one has been working on staging I was able to undo that transaction and fix the setup to only import initial time periods if there aren't any yet.
